### PR TITLE
Clarification of where the value in CSIP86 comes from

### LINKED ISSUE #453: CSIP86 Improve wording
**CSIP86 Main structural division label** `structMap/div/@LABEL` description reads:

>The main division (div) element in the package uses the package ID as the value for the attribute LABEL.

I propose changing it to:

>For the METS document in the root folder, the name of the root folder MUST be used as the value of `@LABEL` (cf. CSIPSTR2). For a METS document in a representation folder, the name of the representation folder MUST be used as the value of `@LABEL` (cf. CSIPSTR10).

I used "MUST" with the belief that the best use of div elements is to mirror the folder structure (see #450).

### PR DESCRIPTION
**Pull request by karinbredenberg**
Added where the value is collectecd from.

Closes #453

### CODE REVIEW

File: profile/CSIP.xml [hunk 1]
```diff
@@ -1026,10 +1026,10 @@
                     </dl>
                 </description>
             </requirement>
-            <requirement ID="CSIP86" REQLEVEL="MUST" EXAMPLES="structMapExample1 structMapExample2">
+            <requirement ID="CSIP86" REQLEVEL="MUST" RELATEDMAT="CSIP1" EXAMPLES="structMapExample1 structMapExample2">
                 <description>
                     <head>Main structural division label</head>
-                    <p xmlns="http://www.w3.org/1999/xhtml">The main division (div) element in the package uses the package ID as the value for the attribute LABEL.</p>
+                    <p xmlns="http://www.w3.org/1999/xhtml">The main division (div) element in the package uses the element named "Content Identification" (CSIP1, mets/@OBJID) as the value for the attribute LABEL.</p>
```
Comment (carlwilson) [inline profile/CSIP.xml]: The package MET document's main division (div) element has an @LABEL attribute value identical to the root mets element's @OBJID attribute.

File: specification/implementation/metadata/mets/structmap/index.md [hunk 2]
```diff
@@ -9,6 +9,8 @@ repeating the structMap element.
 The most crucial requirements for the CSIP mandated structural map are as follows:
 
 - The structMap element has a mandatory attribute @LABEL which has the fixed value of “CSIP structMap”. The @LABEL attribute is used to distinguish the Common Specification mandated structural map occurrence from any other, user-defined, structural maps. As such we can also derive the requirement, that any user-defined structural maps must not use the @LABEL value of “CSIP structMap”;
+- The structure is expressed with division elements (div). The div element can be nested in a tree structure and the implementation used in CSIP gathers the div elements described below in one main structural division element.
```
Comment (carlwilson) [inline specification/implementation/metadata/mets/structmap/index.md]: The package structure is documented using METS division (div) elements which can be arranged in nested tree structures. The CSIP organises it's constituent div elements in a single root structural division element.

File: profile/CSIP.xml (hunk 1 above)
Comment (carlwilson) [reply profile/CSIP.xml]: The package's top-level structural division (div) element must have an @Label attribute value identical to the package identifier, i.e. the same value as the mets/@OBJID attribute.

File: specification/implementation/metadata/mets/structmap/index.md [hunk 3]
```diff
@@ -9,6 +9,8 @@ repeating the structMap element.
 The most crucial requirements for the CSIP mandated structural map are as follows:
 
 - The structMap element has a mandatory attribute @LABEL which has the fixed value of “CSIP structMap”. The @LABEL attribute is used to distinguish the Common Specification mandated structural map occurrence from any other, user-defined, structural maps. As such we can also derive the requirement, that any user-defined structural maps must not use the @LABEL value of “CSIP structMap”;
+- The structure is expressed with division elements (div). The div element can be nested in a tree structure and the implementation used in CSIP gathers the div elements described below in one main structural division element. 
+  - The main structural division use the attribute LABEL with the value being the string value of the content identification element of the package
```
Comment (carlwilson) [inline specification/implementation/metadata/mets/structmap/index.md]: This top level structural division must use the package identifier as the value of its `@LABEL` attribute.

File: profile/CSIP.xml [hunk 4]
```diff
@@ -1026,10 +1026,10 @@
                     </dl>
                 </description>
             </requirement>
-            <requirement ID="CSIP86" REQLEVEL="MUST" EXAMPLES="structMapExample1 structMapExample2">
+            <requirement ID="CSIP86" REQLEVEL="MUST" RELATEDMAT="CSIP1" EXAMPLES="structMapExample1 structMapExample2">
                 <description>
                     <head>Main structural division label</head>
-                    <p xmlns="http://www.w3.org/1999/xhtml">The main division (div) element in the package uses the package ID as the value for the attribute LABEL.</p>
+                    <p xmlns="http://www.w3.org/1999/xhtml">The package's top-level structural division (div) element must have an @LABEL attribute value identical to the package identifier, i.e. the same value as the mets/@OBJID attribute.</p>
```
Comment (koit) [inline profile/CSIP.xml]:
```suggestion
                    <p xmlns="http://www.w3.org/1999/xhtml">The package's top-level structural division (div) element MUST have an @LABEL attribute value identical to the package identifier, i.e. the same value as the mets/@OBJID attribute.</p>
```

File: specification/implementation/metadata/mets/structmap/index.md [hunk 5]
```diff
@@ -9,6 +9,8 @@ repeating the structMap element.
 The most crucial requirements for the CSIP mandated structural map are as follows:
 
 - The structMap element has a mandatory attribute @LABEL which has the fixed value of “CSIP structMap”. The @LABEL attribute is used to distinguish the Common Specification mandated structural map occurrence from any other, user-defined, structural maps. As such we can also derive the requirement, that any user-defined structural maps must not use the @LABEL value of “CSIP structMap”;
+- The package structure is documented using METS division (div) elements which can be arranged in nested tree structures. The CSIP organises it's constituent div elements in a single root structural division element.
```
Comment (koit) [inline specification/implementation/metadata/mets/structmap/index.md]:
```suggestion
- The package structure is documented using METS division (div) elements which can be arranged in nested tree structures. The CSIP organises its constituent div elements in a single root structural division element. 
```

**Comment by koit** [comment]
The text is good now. I only added a typo fix and a capitalisation of "MUST."

**Pull request merged**